pkg/machine/ignition.go

```diff
@@ -431,6 +431,23 @@ Delegate=memory pids cpu io
 		},
 	})
 
+	// Increase the number of inotify instances.
+	files = append(files, File{
+		Node: Node{
+			Group: GetNodeGrp("root"),
+			Path:  "/etc/sysctl.d/10-inotify-instances.conf",
+			User:  GetNodeUsr("root"),
+		},
+		FileEmbedded1: FileEmbedded1{
+			Append: nil,
+			Contents: Resource{
+				// TODO this should be fixed for all vmtypes
```

Comment:
Why not fix it directly for all VM types?  I am a bit afraid that we'd forget about it.

Reply:
this is just some copy&pasta mistake I did, I'll drop the comment